### PR TITLE
📜 Localisation Crusade ⚔ | 5th Chapter

### PR DESCRIPTION
**Pull request by DukeJeans**
German and French translation for mod. Improvements to current English localisations.
____________________________
**Changelog:**

>English improved in xx lines of localisation.
>French and German translations added for xx lines of localisation.

### CODE REVIEW

File: localisation/wc_combat.csv [hunk 1]
```diff
@@ -11,61 +11,61 @@ GetCombatActionDescription_action_increased_defence;[target_duel_enemy.GetFirstN
...
-ability_barkskin_desc;Increases Defence by 50% for 2 turns.\n4 turns cooldown.;;;;;;;;;;;;;x
-ability_absorption;Absorption;;;;;;;;;;;;;x
-ability_absorption_desc;Gives a 50% chance that the next Damage will be fully absorbed.\n3 turns cooldown.;;;;;;;;;;;;;x
-ability_bone_shield;Bone Shield;;;;;;;;;;;;;x
-ability_bone_shield_desc;Gives a 50% chance that the next Damage will be fully absorbed.\n3 turns cooldown.;;;;;;;;;;;;;x
-ability_frost_armor;Frost Armor;;;;;;;;;;;;;x
-ability_frost_armor_desc;Gives a 50% chance that the next Damage will be fully absorbed.\n3 turns cooldown.;;;;;;;;;;;;;x
-ability_shroud_of_shadows;Shroud of Shadows;;;;;;;;;;;;;x
-ability_shroud_of_shadows_desc;Increases Defence by 50% for 2 turns.\n4 turns cooldown.;;;;;;;;;;;;;x
+ability_arcane_blast;Arcane Blast;;Arkane Explosion;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/wc_combat.csv]: I think duel abilities was deleted from the mod since HF?

File: localisation/wc_traits.csv [hunk 2]
```diff
@@ -273,91 +273,91 @@ class_necromancer_7;Great Necromancer VII;;;;;;;;;;;;;x;;;;;;;;;;;;;;
...
 class_necromancer_9_desc;Necromancers use his dark and evil magics to create creatures that teeter on the border of life and death. These (more often mindless than not) undead beings work solely to carry out the necromancer's nefarious will.;;;;;;;;;;;;;x;;;;;;;;;;;;;;
-class_necromancer_10;Legendary Necromancer X;;;;;;;;;;;;;x;;;;;;;;;;;;;;
+class_necromancer_10;Legendary Necromancer X;;Legend�re[From.GetrMale] Nekromant[From.GetinFemale] X;;;;;;;;;;;x;;;;;;;;;;;;;;
 class_necromancer_10_desc;Necromancers use his dark and evil magics to create creatures that teeter on the border of life and death. These (more often mindless than not) undead beings work solely to carry out the necromancer's nefarious will.;;;;;;;;;;;;;x;;;;;;;;;;;;;;
-bunny;Bunny;;Hase;;;;;;;;;;;x;;;;;;;;;;;;;;
-bunny_desc;This character is a small and cute bunny.;;Dieser Charakter ist ein kleines, s��es H�schen.;;;;;;;;;;;x;;;;;;;;;;;;;;
+bunny;Bunny;Lapin;Hase;;;;;;;;;;;x;;;;;;;;;;;;;;
+bunny_desc;This character is a small and cute bunny.;Ce personnage est un mignon et petit lapin.;Dieser Charakter ist ein kleines, s��es H�schen.;;;;;;;;;;;x;;;;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/wc_traits.csv]: do we really need localization for test trait?

File: localisation/wc_traits.csv [hunk 3]
```diff
@@ -273,91 +273,91 @@ class_necromancer_7;Great Necromancer VII;;;;;;;;;;;;;x;;;;;;;;;;;;;;
...
+creature_ghoul;Ghoul;Goule;Ghul;;;;;;;;;;;x;;;;;;;;;;;;;;
+creature_ghoul_desc;Men who have finally been given the grace of undeath are known as ghouls. Varying in size, shape, and form, ghouls are the shocktroopers of every undead mass. Using speed as their advantage in battle, these charging hulks of fallen warriors are every shieldbearer's worst nightmare.;;Diejenigen, denen letztlich die Gnade des Untodes zuteil wurde, werden als Ghuls bezeichnet. Die Ghuls unterscheiden sich in Gr��e, Gestalt und Form und sind die Sto�truppen einer jeden untoten Armee. Indem sie die Geschwindigkeit als ihren Vorteil im Kampf nutzen, sind diese heranst�rmenden Leichname gefallener Krieger der schlimmste Albtraum eines jeden Schildtr�gers.;;;;;;;;;;;x;;;;;;;;;;;;;;
+creature_wraith;Spirit;Esprit;Geist;;;;;;;;;;;x;;;;;;;;;;;;;;
+creature_wraith_desc;This character is an incorporeal spirt.;Ce personnage est un esprit incorporel.;Dieser Charakter ist ein k�rperloser Geist.;;;;;;;;;;;x;;;;;;;;;;;;;;
+brilliant_ruler;Brilliant Ruler;;Brilliante[From.GetrMale] Herrscher[From.GetinFemale];;;;;;;;;;;x;;;;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/wc_traits.csv]: This education was deleted

File: localisation/wc_traits.csv [hunk 4]
```diff
@@ -273,91 +273,91 @@ class_necromancer_7;Great Necromancer VII;;;;;;;;;;;;;x;;;;;;;;;;;;;;
...
+capable_nobleman;Capable Nobleman;;F�higer Edelmann;;;;;;;;;;;x;;;;;;;;;;;;;;
+capable_nobleman_desc;While this person may or may not have a complete education, they are of average ability when it comes to court duties.;;W�hrend diese Person eine vollst�ndige Ausbildung haben kann oder auch nicht, ist sie bei der Aus�bung ihrer Aufgaben von durchschnittlicher Bef�higung.;;;;;;;;;;;x;;;;;;;;;;;;;;
+petty_despot;Petty Despot;;Kleiner Despot;;;;;;;;;;;x;;;;;;;;;;;;;;
+petty_despot_desc;Of little importance, evil, or just plain uneducated in the way of ruling, this person is known to be of poor ability in the way of politics.;;Von geringer Bedeutung, b�se oder einfach nur ungebildet in Fragen der Herrschaft ist diese Person bekannt daf�r, dass sie eine geringe Begabung in Sachen Politik hat.;;;;;;;;;;;x;;;;;;;;;;;;;;
+mastermind_mechanic;Mastermind Mechanic;;Meistermechaniker/;;;;;;;;;;;x;;;;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/wc_traits.csv]: This education was deleted

File: localisation/00_wc_government.csv [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
 uncolonized_government;Uncolonized;Non colonis�;Unkolonisiert;;;;;;;;;;;x
-uncolonized_government_desc;In this territory there is no form of government and it is open for colonization.;Sur ces terres, il n'y a aucune forme de gouvernement ou d��tat. Elles sont ouvertes � la colonisation.;In dieser Provinz ist keinerlei Form von Regierung etabliert, sie kann kolonisiert werden.;;;;;;;;;;;x
+uncolonized_government_desc;There are no rules out here in the wilderness;Sur ces terres, il n'y a aucune forme de gouvernement ou d��tat. Elles sont ouvertes � la colonisation.;In dieser Provinz ist keinerlei Form von Regierung etabliert, sie kann kolonisiert werden.;In dieser Provinz ist keinerlei Form von Regierung etabliert, sie kann kolonisiert werden.;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_wc_government.csv]: This also was deleted

File: localisation/wc_traits.csv [hunk 6]
```diff
@@ -379,86 +379,86 @@ class_shadow_priest_9;Grand Master Shadow Priest IX;;;;;;;;;;;;;x;;;;;;;;;;;;;;
...
 subclass_dark_shaman_desc;This character is a dark shaman: one of those who enslaved and corrupted the elements with power of the void or Fel magic.;;;;;;;;;;;;;x;;;;;;;;;;;;;;
 vampire;Vampire;Vampire;;;;;;;;;;;;x;;;;;;;;;;;;;;
-vampire_desc;This character is a vampire.;;;;;;;;;;;;;x;;;;;;;;;;;;;;
-uncolonized_trait;Uncolonized;;;;;;;;;;;;;x;;;;;;;;;;;;;;
-sleeping_trait;Sleeping;;;;;;;;;;;;;x;;;;;;;;;;;;;;
-sleeping_trait_desc;This character is in a deep sleep.;;;;;;;;;;;;;x;;;;;;;;;;;;;;
+vampire_desc;This character is a vampire.;Ce personnage est un vampire.;;;;;;;;;;;;x;;;;;;;;;;;;;;
+uncolonized_trait;Uncolonized;Non colonis�;;;;;;;;;;;;x;;;;;;;;;;;;;;
+sleeping_trait;Sleeping;Endormi;;;;;;;;;;;;x;;;;;;;;;;;;;;
+sleeping_trait_desc;This character is in a deep sleep.;Ce personnage est dans un profond sommeil.;;;;;;;;;;;;x;;;;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/wc_traits.csv]: uncolonized_trait and sleeping_trait also was deleted

**Comment by MemeWizard** [comment]
> zumbak04 requested review from zumbak04, geffe1 and MemeWizard 11 hours ago

> MemeWizard

Just why

...

Also, emoji in PR names are cancerous

image

File: localisation/00_wc_government.csv [hunk 7]
```diff
@@ -1,23 +1,21 @@
...
 dragon_government;Dragonflight;Vol draconique;Drachenschwarm;;;;;;;;;;;x
-dragon_government_desc;A dragonflight is a hierarchical society of dragons and other dragonkin.;Un vol draconique est une soci�t� hi�rarchique compos�e de dragons et autres draconiens.;Der Drachenschwarm ist eine hierarchische Gesellschaft von Drachen und Drachenartigen.;;;;;;;;;;;x
+dragon_government_desc;The different races of dragons are split up between different dragonflights. Each flight is led by a Dragon Aspect who holds dominion over all the brethren, originally chosen by the Titans. The primary great flights are the blue, black, bronze, green, and red dragonflights. Other lesser flights exist that of which are considered artificial or unnatural, such as the Infinite or Chomatic dragonflights. ;Un vol draconique est une soci�t� hi�rarchique compos�e de dragons et autres draconiens.;Der Drachenschwarm ist eine hierarchische Gesellschaft von Drachen und Drachenartigen.;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_wc_government.csv]: > Each flight is led by a Dragon Aspect who holds dominion over all the brethren

What about independent dragons

<details>

image

</details>

> The different races of dragons are split up between different dragonflights.

or dragons that have other dragons as vassals

<details>

image

</details>

File: localisation/00_zuwc_united_localisation.csv [hunk 8]
```diff
@@ -1003,11 +1003,11 @@ just_fight;Fight;;Kampf;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
 infected_holding;Favourable Conditions for Diseases;;G�nstige Umst�nde f�r Krankheiten;;;;;;;;;;;x
-RULE_DUELS_ON_BATTLEGROUND;Duel on Battleground;;Duell auf dem Schlachtfeld;;;;;;;;;;;x
+RULE_DUELS_ON_BATTLEGROUND;Duel on Battleground;Duel sur le champ de bataille;Duell auf dem Schlachtfeld;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv]: this gamerule was deleted

File: localisation/00_wc_government.csv (hunk 7 above)
Comment (geffe1) [reply localisation/00_wc_government.csv]: Pretty sure uncolonized_government is not a thing anymore, as I recall us removing all of those some time ago.

**Comment by geffe1** [comment]
Making a comment here- but the uncolonized_government is long gone from what I remember.

**Comment by zumbak04** [comment]
> Making a comment here- but the uncolonized_government is long gone from what I remember. And except for that and what Meme found, I have not seen anything else.

I moved it to `coder_resources\_unused\localisation\not_used_strings`

**Comment by MemeWizard** [comment]
Can you also delete unnecessary `;;;;;;;;;;;;;;` after `x` in `wc_traits.csv`
and move `00_wc_buildings.csv` from `localisation\customizable_localisation` to `localisation` folder or `coder_resources\_unused\localisation` folder?

**Comment by MemeWizard** [comment]
wcog, wcband, wcduel, wctinker, wctmb, wcjbe events are deleted or turned off, but localization is still here, same with Noblegarden, Lunar, Love is in the Air, Children's Week, Midsummer Fire, Harvest, Brewfest, Hallows End, Pilgrim's Bounty and Winter Veil festivals, Explorers League, old Old Gods followers traits... and God only know what else.

We probably need a separate branch for just sorting out localization files

**Pull request merged**